src/node/api.js

```diff
@@ -72,6 +75,33 @@ function createAPI (node, webhooks, interval=30) {
     }
 }
 
+function getAnonymousNeigbourNodeStats (nodeList) {
+    nodeList.peers.forEach(function(peer) {
```

Comment:
Best approach here would be using .map() to create a list of json clones of peer.data and changing ip/isAnonymous accordingly on the new objects...